optimize/backend/src/main/java/io/camunda/optimize/service/importing/BackoffImportMediator.java

```diff
@@ -142,62 +131,4 @@ private void calculateNewDateUntilIsBlocked() {
     final long sleepTime = idleBackoffCalculator.calculateSleepTime();
     logger.debug("Was not able to produce a new job, sleeping for [{}] ms", sleepTime);
   }
-
-  private List<String> readExcludedIdsFromConfig() {
```

Comment:
can we make sure that the excluded tenants get removed from the config file when we clear that up in a later PR?

Reply:
Linked to in [config ticket](https://github.com/camunda/camunda-optimize/issues/13375)